### DISCUSSION TITLE
Issues on Windows 10: 1. Installation 2. Blank Screen

### DISCUSSION THREAD

**Discussion by LHSequalsRHS** [flxzt/rnote#1105] [Q&A]
I recently heard about rnote on Reddit and thought I will give it a try. I am using Windows 10 with Avast AV Pro.  For reference, xournal++, OneNote, and StylusLabs works on my laptop. 

While downloading the installer Edge browser warns about potential safety. Windows then warns when installing the software. After installing it with admin permission I ran the software, again with admin, and I received a blank white screen. 

I am not sure what to do next. Thanks.

**Comment by flxzt** [comment]
Sounds a bit like your Anti-Virus software blocks the app?
Unfortunately it can happen that a false-positive is reported. (see #715).

You shouldn't need to run it as admin, but you may need to add it as an exception in avast.

**Comment by LHSequalsRHS** [comment]
I added it as an exception before running it as an admin. Here are the screenshots of the problem:
![RNOTE](https://github.com/flxzt/rnote/assets/25143553/ab0ba696-34e0-45d6-87f8-b761847a52af)
![RNOTE1](https://github.com/flxzt/rnote/assets/25143553/14d9c729-2ff1-40dd-b7f5-e8596b1cf3ea)

**Reply by Doublonmousse** [reply]
A graphical issue with gtk4 seems more likely here ..
What is your graphics card + driver version ? 
Maybe we can find some related issue/fix on the gtk gitlab from that

**Reply by LHSequalsRHS** [reply]
> A graphical issue with gtk4 seems more likely here .. What is your graphics card + driver version ? Maybe we can find some related issue/fix on the gtk gitlab from that

Hey,

So this is my graphics card and driver version...
![image](https://github.com/flxzt/rnote/assets/25143553/e1a1fe73-f9ec-4e77-bca5-7c122388001a)

**Reply by Doublonmousse** [reply]
Okay ... https://gitlab.gnome.org/GNOME/gtk/-/issues/6401 seems like an upstream issue that may be fixed.
If you have a igpu + nvidia card combo, maybe try to force the the igpu to render the application.

 If not I can also try to compile rnote with a newer version of gtk that might work (we'll probably upgrade the gtk version at some point in next releases anyway though)

**Reply by LHSequalsRHS** [reply]
This solution worked. Thanks!
![image](https://github.com/flxzt/rnote/assets/25143553/49c4b181-c11d-4f8b-ac91-2abc294d47d7)

**Comment by mahdihasan2006** [comment]
I have the same problem with the latest version (v0.10.2) of Rnote in Windows 10. So, I am using previous version (v0.10.1). I don't have a separate GPU.

**Reply by Doublonmousse** [reply]
That's weird... It doesn't seem like there's a lot of commits that'd change behavior between the two versions (apart from potential different gtk versions and potential build issues related to windows).

The latest version does work on my old windows 10 machine so at least it's not a general issue for all windows 10 users.

I think i'd be good to retest this with the next release (if we release with gtk 4.14).

Updating graphics drivers may be a good idea
